packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -84,6 +84,21 @@
 +    - effect: "NoSchedule"
 +      operator: "Exists"
 +
++rkeIngressNginx:
++  enabled: false
++  metricsPort: 10254
++  component: ingress-nginx
++  clients:
++    port: 10015
++    useLocalhost: true
++    tolerations:
++      - effect: "NoExecute"
++        operator: "Exists"
++      - effect: "NoSchedule"
++        operator: "Exists"
++    nodeSelector:
++      node-role.kubernetes.io/worker: "true"
```

Comment:
Just to confirm; this means that the ingress-controller cannot exist on non-worker nodes? Where does this requirement come from?

Reply:
I created an RKE2 cluster with 1.20 and found it on the ingress-nginx-controller Deployment

Reply:
Do you mean RKE1?

Reply:
yes, RKE1 cluster.

Reply:
the ingress-nginx Daemonset on RKE1 has the following nodeAffinity

```
    spec:
      affinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: beta.kubernetes.io/os
                operator: NotIn
                values:
                - windows
              - key: node-role.kubernetes.io/worker
                operator: Exists
```